packages/zoe/src/contracts/auction/index.js

```diff
@@ -133,17 +152,15 @@ const start = zcf => {
     };
 
     const customProperties = getSessionDetails();
-    return zcf.makeInvitation(performBid, 'bid', customProperties);
+    return zcf.makeInvitation(
+      performBid,
+      'bid',
+      customProperties,
+      BidProposalShape,
```

Comment:
wondering whether it's always worth defining these constants that will be used in exactly one place.

Reply:
They might be handy for clients to use. (recall our discussion of `client-utils.js` things to accompany contracts).

Reply:
> They might be handy for clients to use. (recall our discussion of client-utils.js things to accompany contracts).

What discussion is that? Sounds interesting.

Reply:
Things like

 - utilities to do AMM calculations in a web UI
 - contract keyword constants or helper functions for constructing offers?
 - governed parameter utilities (several contracts have a `params.js` file that's useful for callers of `startInstance`)

Reply:
https://github.com/Agoric/agoric-sdk/pull/7091

Though note that the production shapes won't work for "remotables" in clientSupport because they use BoardRemotes. Maybe eventually we'll want to get rid of BoardRemotes and leave that to the marshaller.

Reply:
I leaving as separate constant for now. Resolving.